proto/src/determined/trial/v1/trial.proto

```diff
@@ -7,9 +7,55 @@ import "google/protobuf/struct.proto";
 import "determined/experiment/v1/experiment.proto";
 import "determined/checkpoint/v1/checkpoint.proto";
 import "google/protobuf/timestamp.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+
+message TrialValidation {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+      json_schema: {
+      required: ["batch_number", "start_time", "state", "searcher_metric"]
```

Comment:
I imagine end_time isn't included so this can reference undone validations, in which case we should remove searcher_metric too, right?

Reply:
yeah. for the use of this specific pr though validations would always be in completed state. but if we use this elsewhere we should exclude both end_time and seracher_metric